kustomize/data_source_kustomization_build.go

```diff
@@ -42,16 +54,39 @@ func dataSourceKustomization() *schema.Resource {
 	}
 }
 
+type kustomizeOptions struct {
+	loadRestrictor string
+}
+
+func getKustomizeOptions(d *schema.ResourceData) (k kustomizeOptions) {
```

Comment:
This function is probably better located in the same file as runKustomizeBuild.

Reply:
Makes sense